packages/web/services/technology.js

```diff
@@ -98,6 +117,26 @@ export const getTechnologies = async (params = {}) => {
 	return response.data;
 };
 
+/**
+ * Fetches technologies to curate
+ *
+ * @param {object} options Optional params
+ * @param {boolean} [options.embed] Response with embed.
+ * @returns {Array} The technologies.
+ */
+export const getTechnologiesToCurate = async (options = { embed: true }) => {
+	const response = await apiGet('reviewer/technologies', { ...options });
+
+	if (response.status !== 200) return [];
+
+	const { data, headers } = response;
+
+	const totalPages = headers['X-Sabia-Total-Pages'];
+	const totalItems = headers['X-Sabia-Total'];
```

Comment:
Dá pra colocar isso aqui lá no enum da API, o que acha?

```suggestion
	const totalPages = headers[apiEnum.TOTAL_PAGES];
	const totalItems = headers[apiEnum.TOTAL_ITEMS];
```